Player/IMediaElement.cs

```diff
@@ -54,6 +57,8 @@ public interface IMediaElement
 
          event  EventHandler<MouseButtonEventArgs> MouseLeftButtonUp;
 
+         event EventHandler<DateTimeArgs> SyncPointPlayed;
```

Comment:
SendEvent is called from withing SyncPointPlayed handler and is not visible by imediaelement implementor. how the event will be raised otherwised?